docs/code/demo.py

```diff
@@ -6,19 +6,19 @@
 @app.task('daily')
 def do_daily():
     "This function runs once a day"
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).